packages/gestalt/src/RadioGroup.js

```diff
@@ -0,0 +1,64 @@
+// @flow strict
+import { type Node } from 'react';
+import Fieldset from './Fieldset.js';
+import Flex from './Flex.js';
+import RadioGroupButton from './RadioGroupButton.js';
+import { RadioGroupContextProvider } from './RadioGroupContext.js';
+
+type Props = {|
+  /**
+   * The individual RadioGroup.RadioButtons for this group
```

Comment:
Suggestion. Can we add the same SlimBanner content into the description on RadioButton so engs see the info?

And yes the info we'll be redundant but we can override the description for RadioButton easily so it's not duplicated in the docs. I feel passing the info to VScode can be helpful as well.